### ISSUE TITLE
Collect HTTP Server metrics from explicit endpoints

### ISSUE THREAD

**Issue by dangxunb** [vert-x3/vertx-dropwizard-metrics#47]
Hi all,
I tried to configure vertx-dropwizard-metrics with the following config:

``` javascript
{
  "enabled": true,
  "jmxEnabled": true,
  "monitoredServerUris": [
    {
      "value": "/hello"
    }
  ]
}
```

JMX Metrics up nice but it collect information from any endpoint. Is this the expected behavior? I just want to collect information from only one endpoint.

My vert.x version is 3.3.3, I use Jprofiler 9.2 to monitor MBean data.

**Comment by vietj** [comment]
can you explain the issue you have ?

**Comment by dangxunb** [comment]
I want to monitor only one endpoint (/hello) with the above config but it collect metrics from all endpoint

**Comment by pendula95** [comment]
This should is also discussed in #79 and should be fixed by #80. I hope this PR will be merged soon as we required it also.
